frontend/src/utils/Copy.ts

```diff
@@ -2,4 +2,24 @@
 const TRANSLATION_PAGE_TOOL_TIP_COPY =
   "Your story translation is pending review. You can edit your translation and leave comments once a reviewer has given feedback.";
 
-export { TRANSLATION_PAGE_TOOL_TIP_COPY };
+const REVIEW_PAGE_APPROVE_ALL_CONFIRMATION =
+  "Every line (including lines with the “Action” status) will be marked as “Approved”. To officially finish the translation, click “Submit Translation” afterwards.";
+
+const REVIEW_PAGE_APPROVE_LAST_LINE_CONFIRMATION =
+  "By marking this last line as “Approved”, all lines will be approved for this translation. To officially finish the translation, click “Submit Translation” afterwards.";
+
+const TRANSLATION_PAGE_SEND_FOR_REVIEW_CONFIRMATION =
+  "Once you send your translation for review, you wont be able to edit the translation or leave comments until a reviewer reviews your translation. This action cannot be undone. Please make sure that there are no mistakes.";
+
+const REVIEW_PAGE_BUTTON_MESSAGE = "I'm sure, mark all as approved";
+
+const TRANSLATION_PAGE_BUTTON_MESSAGE = "I'm sure, send for review";
+
+export {
```

Comment:
nit: instead of exporting everything at the end, export as you go. Eg.
`export const REVIEW_PAGE_APPROVE_LAST_LINE_CONFIRMATION =...`